### PR TITLE
Change jumpy version

### PR DESCRIPTION
**Pull request by sshepel**
Change beta suffix to snapshot, to be able to upload snapshots.
Update nd4j-native dependency version, to be able to test latest changes from snapshots.

### CODE REVIEW

File: jumpy/pom.xml [hunk 1]
```diff
@@ -4,12 +4,12 @@
     <groupId>org.deeplearning4j</groupId>
     <artifactId>dl4j</artifactId>
     <packaging>jar</packaging>
-    <version>1.0.0-beta</version>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (agibsonccc) [inline jumpy/pom.xml]: Get rid of the url and change the artifactId to jumpy ?

Comment (sshepel) [reply jumpy/pom.xml]: Sorry, which URL?

Comment (sshepel) [reply jumpy/pom.xml]: Ah, I think I found URL :)

**Comment by sshepel** [comment]
@farizrahman4u let me try to do that...

**Comment by sshepel** [comment]
@farizrahman4u I have updated the PR with requested changes please verify...

**Comment by farizrahman4u** [comment]
`jumpy-1.0.0-beta.jar`-> `jumpy-1.0.0-SNAPSHOT.jar`

**Comment by sshepel** [comment]
> jumpy-1.0.0-beta.jar-> jumpy-1.0.0-SNAPSHOT.jar

Ah, yes, let me fix that...

**Comment by sshepel** [comment]
@farizrahman4u fixed.

**Comment by farizrahman4u** [comment]
LGTM, thanks.

**Comment by sshepel** [comment]
Thank you @farizrahman4u.

**Pull request merged**